mps_youtube/commands/play.py

```diff
@@ -92,7 +84,21 @@ def play(pre, choice, post=""):
                 streams.preload(g.model[chosen + 1], override=override)
 
         try:
-            play_range(songlist, shuffle, repeat, override)
+            if not config.PLAYER.get or not util.has_exefile(config.PLAYER.get):
+                g.message = "Player not configured! Enter %sset player <player_app> "\
+                    "%s to set a player" % (c.g, c.w)
+                return
+            else:
+                if config.PLAYER.get.endswith("mpv.exe"):
```

Comment:
using .exe in the endswith() calls breaks playback on linux.

We could change these if statements to if "mpv"/"mplayer" in config.PLAYER.get and it should work for most people unless you put your mpv binary in a folder called mplayer or vice versa.